test/components/allele.js

```diff
@@ -1,28 +0,0 @@
-import AlleleView from '../../src/code/components/allele';
```

Comment:
I couldn't find uses of this anywhere in the codebase, so I removed it and the tests that referred to it